### PR TITLE
move cleanup process to a post-task

### PR DESCRIPTION
**Pull request by morenod**
Move cleanup process to a post-task on osde2e and rosa jobs.

After this merge, clusters will be preserved by default. Cleanup process will be only executed if CLEANUP_CLUSTERS var is true

### CODE REVIEW

File: jjb/dynamic/scale-ci-ms-osde2e.yml [hunk 1]
```diff
@@ -232,12 +226,57 @@
             - ERROR
             - WARNING
             - DEBUG
+      - bool:
+          default: true
```
Comment (dry923) [inline jjb/dynamic/scale-ci-ms-osde2e.yml:230]: @morenod Shouldn't this be false if our default behavior is not to cleanup the clusters? Same with rosa one below

Comment (morenod) [reply jjb/dynamic/scale-ci-ms-osde2e.yml]: yep, updated on both rosa and osde2e

**Review by dry923 — APPROVED** [review]
LGTM

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**